### DISCUSSION TITLE
課題13についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#53] [Q&A]
課題13では、並んでいる数字を右の値と比べていましたが、これが左の値と比べる際は、「j – 1」となる考え方で合っていますでしょうか？
・「j + 1」：右と比べる
・「j – 1」：左と比べる

**Comment by nsc-kiwamu** [comment]
そうですね、j-1での比較となります。
あとはfor文の開始、終了条件が変わるところがわかれば完璧ですね。
